internal/dataplane/parser/translate_httproute.go

```diff
@@ -232,7 +232,9 @@ func generateKongRoutesFromHTTPRouteMatches(
 		return []kongstate.Route{r}, nil
 	}
 
-	// TODO: implement query param matches (https://github.com/Kong/kubernetes-ingress-controller/issues/2778)
+	// Kong starts to support query parameter match since 3.4.1 with expression router.
```

Comment:
```suggestion
```
the same rationale as in the previous comment

Reply:
removed the words about Kong version.